### ISSUE TITLE
Sometimes generates fully-split maps

### ISSUE THREAD

**Issue by patrakov** [Frunit/qpop.js#1]
Extreme example:
![image](https://user-images.githubusercontent.com/2943163/104821180-ee681980-585b-11eb-93ef-76d32fefdc9d.png)

Each species was either confined to its own region by mountains and water, or shared it with one other species. This makes the game totally non-interesting in the "until the bitter end" setting.

**Comment by patrakov** [comment]
Funny fact: just after reporting this bug, an earthquake happened - but still, there is no possibility for a bitter end, too many populated islands.

![image](https://user-images.githubusercontent.com/2943163/104821428-9f22e880-585d-11eb-8255-4d6b55a19114.png)

**Comment by Frunit** [comment]
Sorry for the late answer!

I know what you mean and I also think that it is annoying when that happens. Earthquakes or a lower sea level from the "global warming" or "comet" catastrophes can help. This is the behaviour of the original game, but I will think about how to implement a somewhat sensible option to ensure a land connection to each individual.

What should happen, if the player actively places the first individual on an island? It seems weird to raise some land "magically" to ensure the connection. Or should comets and human bases take care not to block an existing path?
Well, as I said, I will think about it, but any comments are welcome!

**Comment by patrakov** [comment]
I guess, a somewhat "natural" option would be to increase the frequency of quakes somewhat if a path between two individuals does not exist. And to increase it significantly if a player could not use all of the steps.

**Comment by Frunit** [comment]
Yes, that might be a good idea! I will think a bit about it and how to implement it. Thank you!
